dashboard/public/pixelation/pixelation.html

```diff
@@ -67,7 +67,7 @@
       <input type="radio" name="binHex" value="hex" onchange="binToHex(); drawGraph();">
     </label>
   </div>
-  <textarea id="pixel_data" oninput="drawGraph();" autocapitalize="false" autocorrect="false" autocomplete="false" spellcheck="false"></textarea>
+  <textarea id="pixel_data" oninput="drawGraph();" onkeydown="drawGraph();" autocapitalize="false" autocorrect="false" autocomplete="false" spellcheck="false"></textarea>
```

Comment:
Likewise, if this fires twice in newer browsers consider a `debounce` or `throttle`.